### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by xukai92**
KMM test is now fixed. There was also a bug in the KLIEP test (missing `i` for creating the names of plots)

### CODE REVIEW

File: src/kmm/jump.jl [hunk 1]
```diff
@@ -9,15 +9,11 @@ function _densratio(x_nu, x_de, dre::KMM, optlib::Type{JuMPLib})
...
-  # warn user that original convex problem formulation
-  # does not consider regularization parameter
-  iszero(λ) || @warn "λ parameter ignored when optlib=JuMPLib"
-
   # number of numerator and denominator samples
   m′, m = length(x_nu), length(x_de)
 
   # constants for objective and constraints
-  K = gaussian_gramian(x_de, x_de, σ=σ)
+  K = gaussian_gramian(x_de, x_de, σ=σ) + λ * I
```
Comment (xukai92) [inline src/kmm/jump.jl]: We can do the same stablization here. And this is necessary to make the method work.

File: test/kliep.jl [hunk 2]
```diff
@@ -1,5 +1,5 @@
 @testset "KLIEP -- $optlib" for optlib in [OptimLib, ConvexLib]
-  for (pair, rtol) in [(pair₁, 2e-1), (pair₂, 4e-1)]
+  for (i, pair, rtol) in [(1, pair₁, 2e-1), (2, pair₂, 4e-1)]
```
Comment (xukai92) [inline test/kliep.jl]: It is needed in Line 20

File: src/kmm/jump.jl (hunk 1 above)
Comment (juliohm) [reply src/kmm/jump.jl]: Awesome, thanks for the fix.

File: test/kliep.jl (hunk 2 above)
Comment (juliohm) [reply test/kliep.jl]: Perhaps write it as `enumerate([(pair1, 2e-1), (pair2, 4e-1)])` instead?

File: test/kmm.jl [hunk 3]
```diff
@@ -1,22 +1,21 @@
 @testset "KMM -- $optlib" for optlib in [JuliaLib, JuMPLib]
-  for (i, pair) in enumerate([pair₁, pair₂])
+  for (i, pair, rtol) in [(1, pair₁, 2e-1), (2, pair₂, 4e-1)]
```
Comment (juliohm) [inline test/kmm.jl]: I think we can keep enumarate here like suggested above.

File: test/kmm.jl [hunk 4]
```diff
@@ -1,22 +1,21 @@
...
     d_nu, d_de = pair
     Random.seed!(123)
     x_nu, x_de = rand(d_nu, 2_000), rand(d_de, 1_000)
 
     # estimated density ratio
-    σ, B, ϵ, λ = 1.5, Inf, 0.01, 0.001
+    D = [sqrt(DensityRatioEstimation.euclidsq(x, y)) for x in x_nu, y in x_de]
+    σ, B, ϵ, λ = Distributions.mode(D), Inf, 0.001, 0.01
```
Comment (juliohm) [inline test/kmm.jl]: Thanks for fixing it.

**Comment by xukai92** [comment]
I will address them during weekends.

**Comment by xukai92** [comment]
Okay so I replaced explicit indexing with `enumerate` now.

**Pull request merged**